stubs/kafka/structs.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import NamedTuple
+from kafka.errors import KafkaError
+
+class TopicPartition(NamedTuple):
+    topic: str
+    partition: int
+
+class BrokerMetadata(NamedTuple):
+    nodeId: int
+    host: str
+    port: int
+    rack: str
+
+class PartitionMetadata(NamedTuple):
+    topic: str
+    partition: int
+    leader: int
+    replicas: list[int]
+    isr: list[int]
```

Comment:
This form of typing is not available in Python 3.8 which is still supported.
It should be:

```python
from typing import List, NamedTuple

...

    replicas: List[int]
    isr: List[int]
```